lvgl_tft/st7789.c

```diff
@@ -111,37 +116,10 @@ void st7789_flush(lv_disp_drv_t * drv, const lv_area_t * area, lv_color_t * colo
     uint16_t offsety2 = area->y2;
     uint32_t size = lv_area_get_width(area) * lv_area_get_height(area);
 
-#if (CONFIG_LV_TFT_DISPLAY_OFFSETS)
-    offsetx1 += CONFIG_LV_TFT_DISPLAY_X_OFFSET;
-    offsetx2 += CONFIG_LV_TFT_DISPLAY_X_OFFSET;
-    offsety1 += CONFIG_LV_TFT_DISPLAY_Y_OFFSET;
-    offsety2 += CONFIG_LV_TFT_DISPLAY_Y_OFFSET;
-
-#elif (LV_HOR_RES_MAX == 240) && (LV_VER_RES_MAX == 240)
-    #if (CONFIG_LV_DISPLAY_ORIENTATION_PORTRAIT)
-        offsetx1 += 80;
-        offsetx2 += 80;
-    #elif (CONFIG_LV_DISPLAY_ORIENTATION_LANDSCAPE_INVERTED)
-        offsety1 += 80;
-        offsety2 += 80;
-    #endif
-#elif (LV_HOR_RES_MAX == 240) && (LV_VER_RES_MAX == 135)
-    #if (CONFIG_LV_DISPLAY_ORIENTATION_PORTRAIT) || \
-        (CONFIG_LV_DISPLAY_ORIENTATION_PORTRAIT_INVERTED)
-        offsetx1 += 40;
-        offsetx2 += 40;
-        offsety1 += 53;
-        offsety2 += 53;
-    #endif
-#elif (LV_HOR_RES_MAX == 135) && (LV_VER_RES_MAX == 240)
-    #if (CONFIG_LV_DISPLAY_ORIENTATION_LANDSCAPE) || \
-        (CONFIG_LV_DISPLAY_ORIENTATION_LANDSCAPE_INVERTED)
-        offsetx1 += 52;
-        offsetx2 += 52;
-        offsety1 += 40;
-        offsety2 += 40;
-    #endif
-#endif
+    offsetx1 += st7789_x_offset();
+    offsetx2 += st7789_x_offset();
+    offsety1 += st7789_y_offset();
+    offsety2 += st7789_y_offset();
```

Comment:
Is it ok (in terms of speed) to call those functions in comparison of just defines?

Reply:
I didn't made them defines so the user can change the value at runtime.
They're only accessing a global variable.